libraries/botbuilder-dialogs/botbuilder/dialogs/prompts/number_prompt.py

```diff
@@ -30,9 +41,11 @@ def __init__(self, dialog_id: str, validator: object, default_locale: str):
 
         if is_retry and options.retry_prompt is not None:
             turn_context.send_activity(options.retry_prompt)
-        else:
-            if options.prompt is not None:
-                await turn_context.send_activity(options.prompt)
+        # else:
```

Comment:
Could you remove the commented code?

Reply:
Oops. How'd that get back in there. Gone! :)